.gitignore

```diff
@@ -1,5 +1,9 @@
 # See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
 
+.vscode
```

Comment:
The idea of adding this files was to ake the config of the editor the same among developers to make easier for someone that is getting the project.